database/seeds/UsersTableSeeder.php

```diff
@@ -9,8 +11,11 @@ public function run()
     {
         DB::table('users')->insert([
             'name' => 'Administrator',
-            'email' => 'example@gmail.com',
-            'password' => bcrypt('123'),
+            'email' => 'administrator@example.local',
+            'password' => bcrypt('123456'),
```

Comment:
If you'll create mutator once in model, you will never need to do bcrypt() again.